src/components/FilterBlock.tsx

```diff
@@ -157,7 +172,10 @@ const Filter: React.FC<FilterProps> = ({
         {filter?.operator !== "notNull" &&
           (!filter?.operator?.includes("equals") || fieldValues.length > 100 ? (
             <TextInput
-              setFieldValue={(value) => onFilterChange({ ...filter, value })}
+              setFieldValue={(value) => {
+                onFilterChange({ ...filter, value })
+                filter.value=value;
```

Comment:
don't assign values to a property

Reply:
Forgot to remove it when I had been testing. Will remove that!